apps/src/dance/types.ts

```diff
@@ -1,7 +1,18 @@
+import {LevelProperties, ProjectSources} from '../lab2/types';
+
 export type SongData = {
   [key: string]: {
     title: string;
     url: string;
     pg13: boolean;
   };
 };
+
+export interface DanceProjectSources extends ProjectSources {
+  selectedSong?: string;
+}
+
+export interface DanceLevelProperties extends LevelProperties {
+  defaultSong?: string;
+  useRestrictedSongs?: boolean;
```

Comment:
I see `defaultSong` defined [here](https://github.com/code-dot-org/code-dot-org/blob/1fa26a105d16a9520ac7bf25ab0d557c660b81bb/dashboard/app/models/levels/dancelab.rb#L29) but not `useRestrictedSongs`. Is `DanceLevelProperties` different from these properties? Just trying to understand where `useRestrictedSongs` is coming from.